pages/en/developing/assemblyscript-api.mdx

```diff
@@ -761,3 +761,14 @@ The base `Entity` class and the child `DataSourceContext` class have helpers to
 - `getBytes(key: string): Bytes`
 - `getBoolean(key: string): boolean`
 - `getBigDecimal(key: string): BigDecimal`
+
+### Common AssemblyScript Issues
+
+There are certain [AssemblyScript](https://github.com/AssemblyScript/assemblyscript) issues that are common to run into during subgraph development. They range in debug difficulty, however, being aware of them may help. The following is a non-exhaustive list of these issues:
+
+- Be careful with handler computation limits, especially when using the `.pow()` function with large numbers.
+  - As of [`graph-node v0.26.0`](https://github.com/graphprotocol/graph-node/releases/tag/v0.26.0) gas metering was introduced.
+- Required fields are handled differently depending on the `graph-cli` version used.
+  - In older `graph-cli` releases default values are automatically set, but this was changed to require non-nullable fields to be set in [`graph-cli vTODO?`]().
```

Comment:
This was actually an inadvertent codegen issue which didn't exist in oldest versions, but was introduced temporarily. The reason this was reverted back was to support the `upsert` behaviour in Graph Node when you create an entity that already exists. I think in general we should encourage usage of the most recent versions, so I am not sure if we want to phrase this in this way?

Reply:
I think we probably want to be more definitive to explain the current behaviour, with a caveat that in some prior versions of graph-cli this was handled differently

Reply:
removed from here, should this be added somewhere else? But to be clear, currently null fields that are not set have defaults and will not fail to index?

Reply:
sorry I was unclear - null fields don't have defaults, they need to be set (though I think we should ideally catch this during `build` rather than runtime)

Reply:
Do you think these additions fit here? To me it feels somewhat out of place. It's really just assembly script quirks that we have to be cognizant of during subgraph development.